Arrowgene.Ddon.Database/Sql/Core/DdonSqlDbBazaarExhibition.cs

```diff
@@ -0,0 +1,228 @@
+using System.Collections.Generic;
+using System.Data.Common;
+using Arrowgene.Ddon.Shared.Model;
+
+namespace Arrowgene.Ddon.Database.Sql.Core
+{
+    public abstract partial class DdonSqlDb<TCon, TCom, TReader> : SqlDb<TCon, TCom, TReader>
+        where TCon : DbConnection
+        where TCom : DbCommand
+        where TReader : DbDataReader
+    {
+        
+        private static readonly string BazaarExhibitionTableName = "ddon_bazaar_exhibition";
+        
+        private static readonly string[] BazaarExhibitionFields = new string[]
+        { 
+           /* bazaar_id */ "character_id", "sequence", "item_id", "num", "price", "exhibition_time", "state", "proceeds", "expire"
```

Comment:
Why is `/* bazaar_id */` in comments here?

Reply:
The primary key is never included in the fields variable. I added a comment so its known its intentional